reana_server/utils.py

```diff
@@ -511,27 +524,22 @@ def _get_gitlab_hook_id(project_id, gitlab_token):
     """
     gitlab_hooks_url = (
         REANA_GITLAB_URL
-        + "/api/v4/projects/{0}/hooks?access_token={1}".format(project_id, gitlab_token)
+        + "/api/v4/projects/{0}/hooks?per_page=100&access_token={1}".format(
```

Comment:
Perhaps we could introduce a new configuration variable in `config.py` with `REANA_GITLAB_PER_PAGE = 100` so that the value wouldn't be hard-coded?

Reply:
I have introduced `REANA_GITLAB_MAX_PER_PAGE`